script/travis_retry.sh

```diff
@@ -28,18 +28,19 @@ travis_retry() {
   set +euo pipefail
   local result=0
   local count=1
-  while [ $count -le 3 ]; do
+  local max=666
+  while [ $count -le $max ]; do
     [ $result -ne 0 ] && {
-      echo -e "\n${ANSI_RED}The command \"$*\" failed. Retrying, $count of 3.${ANSI_RESET}\n" >&2
+      echo -e "\n${ANSI_RED}The command \"$*\" failed. Retrying, $count of $max.${ANSI_RESET}\n" >&2
     }
     "$@"
     result=$?
     [ $result -eq 0 ] && break
     count=$((count + 1))
     sleep 1
   done
-  [ $count -gt 3 ] && {
-    echo -e "\n${ANSI_RED}The command \"$*\" failed 3 times.${ANSI_RESET}\n" >&2
+  [ $count -gt $max ] && {
+    echo -e "\n${ANSI_RED}The command \"$*\" failed $max times.${ANSI_RESET}\n" >&2
```

Comment:
I was surprised this actually worked. The Intel apt repository is sometimes not available and `sudo apt add-repository` fails after 3 attempts. But trying more often actually works and does not noticably slow down the CI. I am open to have a different `max` here, but `3` is definitely too little :)

Reply:
An ideal magic number would be `42` ^^